src/main/java/seedu/address/model/person/exceptions/PersonNotFoundException.java

```diff
@@ -1,6 +1,6 @@
 package seedu.address.model.person.exceptions;
 
 /**
- * Signals that the operation is unable to find the specified person.
+ * Signals that the operation is unable to find the specified Schedule.
```

Comment:
Did you use reflector for this? =O

Reply:
yes..i'll change it back. that's an unintentional change.

Reply:
nvm i realised i edited it.. i wasn't due to reflector but i edited that wrong file just now